cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
I'm not sure if I explained it clearly enough, so I'll try again.  The test that you modified calls `republish_collection`, but actually we need to test the thing (the trigger) that calls `republish_collection`, which is https://github.com/Connexions/cnx-archive/blob/15f41bbcc3da0a6ad3f972ffc8bc6a84445423b7/cnxarchive/sql/schema/triggers.sql#L150-L152

That's why I'm suggesting that you also change the test here https://github.com/Connexions/cnx-archive/blob/15f41bbcc3da0a6ad3f972ffc8bc6a84445423b7/cnxarchive/tests/test_database.py#L984-L1083 which is actually testing the trigger.